extensions/amp-image-viewer/0.1/amp-image-viewer.js

```diff
@@ -144,7 +143,8 @@ export class AmpImageViewer extends AMP.BaseElement {
   /** @override */
   onLayoutMeasure() {
     if (this.loadPromise_) {
-      this.loadPromise_.then(() => this.measure());
+      this.loadPromise_.then(() => this.measure())
```

Comment:
This is to fix an existing bug where:
If you rotate the orientation after the image has already been zoomed, it rests but does not allow swipe to change carousel slide.